src/lib/libwasm_worker.js

```diff
@@ -113,11 +113,12 @@ addToLibrary({
     // already exists".  So for now, invoke this function from JS side. TODO:
     // remove this in the future.  Note that this call is not exactly correct,
     // since this limit will include the TLS slot, that will be part of the
-    // region between m['sb'] and m['sz'], so we need to fix up the call below.
-    ___set_stack_limits(m['sb'] + m['sz'], m['sb']);
+    // region between wwParams.stackLowAddress and wwParams.stackSize, so we
+    // need to fix up the call below.
+    ___set_stack_limits(wwParams.stackLowAddress + wwParams.stackSize, wwParams.stackLowestAddress);
```

Comment:
Above we have `stackLowestAddress` and here `stackLowAddress` - ?

Reply:
I guess that explains why the tests didn't pass :)